packages/bitcore-node/src/routes/api/block.ts

```diff
@@ -146,6 +147,41 @@ router.get('/before-time/:time', async function(req: Request, res: Response) {
   }
 });
 
+router.get('/:blockId/feeRate', async function(req: Request, res: Response) {
```

Comment:
Nit pick: what if we called this endpoint `feeData` instead of `feeRate`? or just `fee`